app/views/examples/labels-legends-and-headings/index.njk

```diff
@@ -2838,12 +2694,11 @@
     })
   }}
 
-
   <h3 class="govuk-heading-s">isPageHeading: false</h3>
 
   {{ govukDateInput({
     id: 'dob',
```

Comment:
I think this `id` should also be updated? There are a couple of date input components with the id `dob` still.